src/Illuminate/Log/Context/Repository.php

```diff
@@ -311,6 +311,52 @@ public function pushHidden($key, ...$values)
         return $this;
     }
 
+    /**
+     * Checks if the given value is in the stack.
+     *
+     * @param  string  $key
+     * @param  mixed  $value
+     * @param  bool  $strict
+     * @return bool
+     *
+     * @throws \RuntimeException
+     */
+    public function in(string $key, mixed $value, bool $strict = false): bool
```

Comment:
I wonder if `in` is too generic. Feels like you are checking that the key is in the context.

Wonder if `Context::stackContains` might be better.

That also has symmetry with the `Collection::contains` method. It could accept a closure so that you can do more complex checks if needed.


```
Context::stackContains($stack, $value);

Context::stackContains($stack, function ($value) {
    // ...
});
```

That way we don't need the `strict` flag either, as the closure gives you that escape hatch.

Reply:
I would indeed be more inclined to go this way if the underlying stack was a collection, but it seems to be a deliberate choice to have this as an array. As the rest of the functions, such as `has()` do an `array_key_exists()`, I would say the current implementation is more true to the current implementation of Context in general.

Not necessarily shooting this down, but I feel this is more something for down the line if (if ever) Context receives a bit of a bigger update.